spec/controllers/steps_controller_spec.rb

```diff
@@ -2,11 +2,17 @@
 
 RSpec.describe StepsController do
   context "when no applications have been started" do
-    it "redirects to the homepage" do
+    it "redirects to the homepage on a GET" do
       get :index
 
       expect(response).to redirect_to(root_path)
     end
+
+    it "redirects to the homepage on a PUT" do
+      put :update, params: { id: "" }
```

Comment:
when would this request to the `StepsController` happen IRL?

Reply:
Never would.